cypress/integration/group2/myworkflows.ts

```diff
@@ -444,6 +444,23 @@ describe('Dockstore my workflows', () => {
     });
   });
 
+  describe('Should require default version to publish', () => {
+    it('should not be able to publish with no default version', () => {
+      cy.visit('/my-workflows/github.com/A/l');
+      cy.get('#publishButton').should('contain', 'Unpublish').should('be.visible').click();
+      cy.get('#publishButton').should('contain', 'Publish').should('be.visible').click();
+      cy.get('[data-cy=close-dialog-button]').should('be.visible').click();
+      cy.get('#publishButton').should('contain', 'Publish').should('be.visible');
+    });
+    it('should be able to publish after setting default version', () => {
+      goToTab('Versions');
+      cy.contains('button', 'Actions').should('be.visible').click();
+      cy.get('[data-cy=set-default-version-button]').should('be.visible').click();
+      cy.wait(1000);
```

Comment:
See if the wait could be removed, try replacing it with an assertion that checks if the default version has been switched. if that doesn't work, leave it.